CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 2022-09-08 Contract Checks! 
+| What's new | |
+| :--------- | :-: |
+| We have introduced checks for contracts! Each contract run task includes a build check and an operation check. This latest release will ensure that no contract (downstream) variants are affected by changes made to the source variant’s subgraphs or by changes made to the contract filtering configuration.|
```

Comment:
Did you mean to say `Each contract runs a build check and an operation check` or did you mean `Each contract task includes ...`? :)